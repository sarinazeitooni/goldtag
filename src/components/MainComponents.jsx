import {Fragment} from "react";
import MainSwipper from './swiper/mainswipper';
import Features from './features/features';
import Goldtags from './goldtags/goldtags';
import Category from './category/category';
import Social from './social/social';
function MainComponents() {
    return (
        <Fragment>
            <MainSwipper/>
            <Features/>
            <Goldtags/>
            <Category/>
            <Social/>
        </Fragment>
    );
}
export default MainComponents;