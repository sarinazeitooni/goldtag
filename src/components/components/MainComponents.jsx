import MainSwipper from './swiper/mainswipper';
import Features from './features/features';
import Goldtags from './goldtags/goldtags';
import Category from './category/category';
import Social from './social/social';
function MainComponents() {
    return (
        <div>
            <MainSwipper/>
            <Features/>
            <Goldtags/>
            <Category/>
            <Social/>
        </div>
    );
}
export default MainComponents;