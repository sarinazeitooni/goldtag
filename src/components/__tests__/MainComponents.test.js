import React from 'react';
import { shallow } from 'enzyme';
import MainComponents from "../MainComponents";
import MainSwipper from "../swiper/mainswipper";
import Goldtags from "../goldtags/goldtags";
import Category from "../category/category";
import Features from "../features/features";
import Social from "../social/social";
describe("MainComponents component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<MainComponents/>)
        expect(container).toMatchSnapshot();
    });
    it("it should render MainSwipper",()=>{
        const container = shallow(<MainComponents/>);
        expect(container.find(<MainSwipper/>)).toBeTruthy();
    });
    it("it should render Goldtags",()=>{
        const container = shallow(<MainComponents/>);
        expect(container.find(<Goldtags/>)).toBeTruthy();
    });
    it("it should render Category",()=>{
        const container = shallow(<MainComponents/>);
        expect(container.find(<Category/>)).toBeTruthy();
    });
    it("it should render Features",()=>{
        const container = shallow(<MainComponents/>);
        expect(container.find(<Features/>)).toBeTruthy();
    });
    it("it should render Social",()=>{
        const container = shallow(<MainComponents/>);
        expect(container.find(<Social/>)).toBeTruthy();
    });
});