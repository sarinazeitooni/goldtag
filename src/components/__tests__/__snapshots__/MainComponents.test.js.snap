// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainComponents component it should render a div 1`] = `ShallowWrapper {}`;
